verification/policy.go

```diff
@@ -47,107 +46,85 @@ func isPresent(val string, values []string) bool {
 // if any rule is violated, returns an error
 func ValidatePolicyDocument(policyDoc *PolicyDocument) error {
 	// Constants
-	wildcardScope := "*"
+	wildcard := "*"
 	supportedPolicyVersions := []string{"1.0"}
 	supportedVerificationPresets := []string{"strict", "permissive", "audit", "skip"}
 	supportedTrustStorePrefixes := []string{"ca", "signingservice"}
 
 	// Validate Version
 	if !isPresent(policyDoc.Version, supportedPolicyVersions) {
-		return errors.New("Version '" + policyDoc.Version + "' is not supported")
+		return fmt.Errorf("version '%s' is not supported", policyDoc.Version)
```

Comment:
nit: `'%s'` can be replaced by `%q`, which automatically quotes strings.

Reply:
Addressed